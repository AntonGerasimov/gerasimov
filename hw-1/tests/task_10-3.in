-3 111	-54.54
