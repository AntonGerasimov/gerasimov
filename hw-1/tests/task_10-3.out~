-3
-54.540001
o
