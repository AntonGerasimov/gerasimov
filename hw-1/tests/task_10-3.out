-3
-54.500000
o
